src/DependencyInjection/NetzmachtContaoLeafletMetaModelsExtension.php

```diff
@@ -29,13 +31,19 @@ final class NetzmachtContaoLeafletMetaModelsExtension extends Extension
      */
     public function load(array $configs, ContainerBuilder $container): void
     {
-        $loader = new XmlFileLoader(
+        $xmlLoader = new XmlFileLoader(
             $container,
             new FileLocator(__DIR__.'/../Resources/config')
         );
 
-        $loader->load('listeners.xml');
-        $loader->load('services.xml');
+        $xmlLoader->load('listeners.xml');
+        $xmlLoader->load('services.xml');
+        
+        $ymlLoader = new YamlFileLoader(
+            $container,
+            new FileLocator(__DIR__.'/../Resources/config')
+        );
+        $ymlLoader->load('hooks.yml');
```

Comment:
Please use the existing `listeners.xml` to register new hook listeners.

Reply:
Moved definition to `listeners.xml`